test/connectivity/scripts/connectivityTest.sh

```diff
@@ -58,33 +68,73 @@ function prepare_test_from_artifacts() {
     
 }
 
-function print_logs() {
+function print_deployment_logs() {
+    print_highlighted_message 'LOGS FROM IOTEDGED'
+    journalctl -u iotedge -u docker --since "$test_start_time" --no-pager || true
+
+    print_highlighted_message 'EDGE AGENT LOGS'
+    docker logs edgeAgent || true
+}
+
+function print_test_run_logs() {
     local ret=$1
-    local test_end_time=$2
-    local elapsed_seconds=$3
 
-    elapsed_time="$(TZ=UTC0 printf '%(%H:%M:%S)T\n' "$elapsed_seconds")"
-    print_highlighted_message "Test completed at $test_end_time, took $elapsed_time."
+    print_highlighted_message 'test run exit code=$ret'
+    print_highlighted_message 'Print logs'
+    print_highlighted_message 'testResultCoordinator LOGS'
+    docker logs testResultCoordinator || true
 
     if (( ret < 1 )); then
         return;
     fi
 
-    print_highlighted_message 'Print logs'
     print_highlighted_message 'LOGS FROM IOTEDGED'
     journalctl -u iotedge -u docker --since "$test_start_time" --no-pager || true
 
-    print_highlighted_message 'EDGE AGENT LOGS'
+    print_highlighted_message 'edgeAgent LOGS'
     docker logs edgeAgent || true
```

Comment:
Why ```|| true```?

Is this to prevent a failing exit code where module doesn't exist?

Reply:
yes.